scripts/new_upstream_snapshot.py

```diff
@@ -333,14 +334,30 @@ def get_bugs_fixed_devel():
             yield line.split("LP: #")[1].strip()
 
 
+def increment_major_version(original: str) -> str:
```

Comment:
bikeshed, this is increment_major_minor_version  not just `increment_major_version`.